wazuh/wazuh-kibana-app
# Sanitize report's inputs and usernames

## Summary 

Closes #4329

## To test

- Check the reports' folder for a user `$WAZUH_APP_HOME/downloads/reports/<username>` has been automatically renamed to `$WAZUH_APP_HOME/downloads/reports/hash(<username>)`, and that their reports are still inside the new folder.
- Check no report from other users can be obtained.
- Check there are no collisions between files in read/write operations.
- Check the report name is no longer sent from the frontend (check the request using the dev tools).
- Check that usernames with UTF-8 characters gets encoded by the MD5 hash function.
